README.md

```diff
@@ -0,0 +1,9 @@
+Digital Marketplace API client
+=========================
+
+[![Coverage Status](https://coveralls.io/repos/alphagov/digitalmarketplace-apiclient/badge.svg?branch=master&service=github)](https://coveralls.io/github/alphagov/digitalmarketplace-apiclient?branch=master)
+[![Requirements Status](https://requires.io/github/alphagov/digitalmarketplace-apiclient/requirements.svg?branch=master)](https://requires.io/github/alphagov/digitalmarketplace-apiclient/requirements/?branch=master)
+
+## What's in here?
+
+Digital Marketplace Data and Search API clients
```

Comment:
This could do with a bit more info. Maybe just an example and links to the other relevant repos.